src/geo/mercator_coordinate.js

```diff
@@ -139,6 +139,9 @@ class MercatorCoordinate {
      * to transform into `MercatorCoordinate`s.
      *
      * @returns {number} Distance of 1 meter in `MercatorCoordinate` units.
+     * @example
+     * const coord = new mapboxgl.MercatorCoordinate(0, 0, 0.02);
+     * coord.meterInMercatorCoordinateUnits(); // 2.895799898894401e-7
```

Comment:
I've seen a bit of confusion with this method so this might be a good spot for a larger example that does something more. Maybe something like this?

```suggestion
     * // Calculate a new MercatorCoordinate that is 150 meters west of the other coord.
     * const coord = new mapboxgl.MercatorCoordinate(0.5, 0.25, 0);
     * const offsetInMeters = 150;
     * const offsetInMercatorCoordinateUnits = offsetInMeters * coord.meterInMercatorCoordinateUnits();
     * const westCoord = new mapboxgl.MercatorCoordinate(coord.x - offsetInMercatorCoordinateUnits, coord.y, coord.z);
```

Reply:
yes nice one! Added.